src/test/operators/import_csv_test.cpp

```diff
@@ -103,7 +103,7 @@ TEST_F(OperatorsImportCsvTest, EmptyStrings) {
 
 TEST_F(OperatorsImportCsvTest, Parallel) {
   CurrentScheduler::set(std::make_shared<NodeQueueScheduler>(Topology::create_fake_numa_topology(8, 4)));
-  auto importer = std::make_shared<OperatorTask>(std::make_shared<ImportCsv>("src/test/csv/float_int_large.csv"));
+  auto importer = std::make_shared<OperatorTask>(std::make_shared<ImportCsv>("src/test/csv/float_int_large.csv"), true);
```

Comment:
I know you hate them, but this call should take an enum (`enum class CleanupOperators {Yes, No}` e.g.) and not a bool. Reading this line gives no indication what this "true" does.